VogtPayroll/Employee.cs

```diff
@@ -6,68 +6,122 @@ namespace VogtPayroll
+        /// </summary>
+        /// <returns>paid fed tax</returns>
+        private decimal ComputePaidFedTax()
+        {
+            decimal grossPay;
+            decimal paidFedTax;
+            // compute amount of money emp should pay as fed tax. use formula employee paid fed taxes = 30% of gross pay
 
-            Console.WriteLine("What is the employee's payrate?");
-            var empPayRate = Convert.ToDecimal(Console.ReadLine());
+            grossPay = ComputeGrossPay();
 
-            Employee employee = new Employee(empName, empTotalWorkingHours, empPayRate);
+            paidFedTax = grossPay * .30m;
 
-            return employee;
+            return paidFedTax;
 
         }
-
-        public void ComputeGrossPay(Employee employee)
+        #endregion
+
+        #region ComputePaidStateTax
+        /// <summary>
+        /// Computes paid state tax
+        /// </summary>
+        /// <returns>paid state tax</returns>
+        private decimal ComputePaidStateTax()
         {
-            // compute gross pay use formula Employee gross pay = hours worked times hourly payrate
             decimal grossPay;
+            decimal paidStateTax;
+            // compute money which should pay as state tax. use formula emp paid state taxes 10% of gross pay
 
+            grossPay = ComputeGrossPay();
 
-        }
+            paidStateTax = grossPay * .10m;
 
-        public void ComputePaidFedTax()
-        {
-            // compute amount of money emp should pay as fed tax. use formula employee paid fed taxes = 30% of gross pay
-        }
+            return paidStateTax;
 
-        public void ComputePaidStateTax()
-        {
-            // compute money which should pay as state tax. use formula emp paid state taxes 10% of gross pay
         }
-
-        public void ComputeNetPay()
+        #endregion
+
+        #region ComputeNetPay
+        /// <summary>
+        /// Computes net pay
+        /// </summary>
+        /// <returns>net pay</returns>
+        private decimal ComputeNetPay()
         {
+            decimal grossPay;
+            decimal netPay;
+            decimal paidFedTax;
+            decimal paidStateTax;
             // compute net pay. use formula employee net pay = employee gross pay - (paid fed tax + paid state tax)
-        }
 
-        public void PrintEmpInfo()
+            grossPay = ComputeGrossPay();
+            paidFedTax = ComputePaidFedTax();
+            paidStateTax = ComputePaidStateTax();
+
+            netPay = grossPay - (paidFedTax + paidStateTax);
+
+            return netPay;
+
+        }
+        #endregion region
+
+        #region PrintEmpInfo
+        /// <summary>
+        /// Prints employee information
+        /// </summary>
+        /// <param name="employee">employee object passed in</param>
+        public void PrintEmpInfo(Employee employee)
         {
             // emp name
             // emp total working hours
             // emp hourly pay
```

Comment:
I'd usually remove the comments that I had for placeholders. The code itself down below is descriptive enough to read.

Some developers argue that you can get away with rarely having comments if you name your variables/functions well enough.